R/utility.R

```diff
@@ -326,24 +328,19 @@ waitForNodesToComplete <- function(poolId, timeout = 86400) {
 #' getJobResult(jobId = "job-001")
 #' }
 #' @export
-getJobResult <- function(jobId = "", ...) {
-  args <- list(...)
+getJobResult <- function(jobId = "") {
```

Comment:
Remove the empty string

If the user uses getJobResult(), it would just throw any way.

getJobResult <- function(jobId) {

Reply:
I will fix this in getjobresult branch directly as I just merged the PR.